### DISCUSSION TITLE
Can the tabs show as multiple lines

### DISCUSSION THREAD

**Discussion by namphamdev** [zed-industries/zed#7420] [Feature Requests]
Is it possible to have a wrap tabs feature similar to vscode? Like this: https://bobbyhadz.com/blog/wrap-tabs-to-multiple-lines-vscode.

**Comment by SomeoneToIgnore** [comment]
Nothing like this exists in Zed yet, but you can use smth. like https://github.com/zed-industries/zed/pull/7356 for inspiration and submit a PR 🙂

**Comment by m-ar-c** [comment]
But why close the issue ? Can't we reopen it ?

**Comment by m-ar-c** [comment]
Yeah, progress 🥳 #46573.

Pinned tabs can now be shown on their own row.
